subcommands/config/log.go

```diff
@@ -30,18 +30,52 @@ func doConfigLog(cmd *cobra.Command, args []string) {
 		subcommands.LogConfigs(&subcommands.LogConfigsOptions{
 			Limit: listLimit,
 			ListFunc: func() (*client.DeviceConfigList, error) {
-				return api.FactoryListConfig(factory)
+
+				list, err := api.FactoryListConfig(factory)
+				for _, dc := range list.Configs {
+					if len(dc.CreatedBy) > 0 {
+						user, _ := api.UserAccessDetails(factory, dc.CreatedBy)
+						dc.CreatedBy = user.Name
+					}
+				}
+				return list, err
+			},
+			ListContFunc: func(url string) (*client.DeviceConfigList, error) {
+				list, err := api.FactoryListConfigCont(url)
+				for _, dc := range list.Configs {
+					if len(dc.CreatedBy) > 0 {
+						user, _ := api.UserAccessDetails(factory, dc.CreatedBy)
+						dc.CreatedBy = user.Name
+					}
+				}
+				return list, err
```

Comment:
These new lines of code seem to be the same for all functions, so they'd better go into the `LogConfigs`.

If you need to add one more parameter into the `LogConfigsOptions` to make this possible (e.g. `Factory`) - it's better to add that single parameter than copy paste the same 7 lines of code in 6 places.

Reply:
Removed these in update. :smile: